### DISCUSSION TITLE
Some questions about Velaptor

### DISCUSSION THREAD

**Discussion by RafaelOliveira** [KinsonDigital/Velaptor#646] [Q&A]
Hi, I'd like to ask some questions about Velaptor.

Is there a way to scale the game? Something like have a window with 800x600, but a "virtual window" with 400x300, scaled up (or could be bigger, scaled down). It's useful for games with pixel art.
And, is there support for gamepads? I searched for these two things but I didn't find in the source code (I was looking the docs some days ago, but it's offline now).

**Comment by CalvinWilkinson** [comment]
@RafaelOliveira I first off want to apologize for not answering you much earlier!!

I thought that the built-in GitHub notification system would let me know that I have some new conversation content, but it does not let me know this.  I will look into getting notified or checking this often.

Just a quick heads up though, we do have a discord community where most of the conversations happen.  You are welcome to join!!
Click [here](https://discord.gg/qewu6fNgv7)

**Q:** Is there a way to scale the game?
**A:** Just to be clear, are you saying that you essentially want the ability to "zoom" or stretch the texture into a specified amount of space, which would force the graphics to "look" pixelated?

**Q:** Is there support for gamepads?
**A:** Currently, there is not.  But I definitely will be adding this in the future.  But if you wanted to, you could absolutely add this yourself. Though this is probably not ideal and I fully understand that.


Just a heads up, Velaptor is always being worked on and moving forward, but if you want to help contribute in any way to help speed things up, we are taking contributors and have already picked up some contributors that are helping out!!  If not, no big deal.  Come join the community and say hi!!

**Comment by CalvinWilkinson** [comment]
@RafaelOliveira 

I just wanted to give you an update on Velaptor and what we have coming up. 

Currently, Velaptor now works on Windows, Linux and macOS so we have full desktop cross-platform support coming soon.  The next upcoming release `v1.0.0-preview.48` will bring this feature.  Exciting!!

The next feature I will be working on is gamepad support which will come in the in release `v1.0.0-preview.49`.

The docs website now has a dedicated blog section.  We post about various topics as well as every single release.
You can find that [here](http://localhost:3000/blog/tags/velaptor)

If you have any other questions, please don't hesitate to reach out.
